kong/client_test.go

```diff
@@ -32,6 +32,19 @@ func TestKongStatus(T *testing.T) {
 	assert.NotNil(status)
 }
 
+func TestKongConfig(t *testing.T) {
+	RunWhenDBMode(t, "off")
+	assert := assert.New(t)
+
+	client, err := NewTestClient(nil, nil)
+	assert.NoError(err)
```

Comment:
```suggestion
	require.NoError(t, err)
```

Reply:
I used "assert" to be coherent with all the other tests of this file.

Reply:
I get your point. OTOH though why copy broken patterns :)

Reply:
Why broken? isn't just the same as using the approach you suggested?

Reply:
When `err` is not `nil` then most likely the `client` is `nil` (I'd suggest to change the check below to similarly use `require` as well).

In such a case any call to `client`'s methods would most likely panic. With `require` you wouldn't get a panic but a precise error message and the test would stop at this point.

Reply:
Gotcha, I'm gonna fix it

Reply:
fixed